src/main/webapp/script.js

```diff
@@ -82,4 +84,14 @@ async function getChart() {
 
 function toRangeStringRepresentation(range) {
   return range.getInclusiveStart() + " to " + range.getExclusiveEnd();  
-}
\ No newline at end of file
+}
+
+function getRangeExclusiveEnd(rangeString) {
+  rangeString.trim();
+  return Number(rangeString.substring(rangeString.indexOf(',') + 1, rangeString.length - 1));
+}
+ 
+function getRangeInclusiveStart(rangeString) {
+  rangeString.trim();
+  return Number(rangeString.substring(1, rangeString.indexOf(',')));
+}
```

Comment:
Is this the recommended way to do substring? I feel this is a little bit time-consuming:(

Reply:
Talked offline with Rick, fine for now

Reply:
Agreed, this seems messy. Is it possible to do: JSON.parse() to convert the string to the actual object?